### PR TITLE
Wrapper script panorama

### PR DESCRIPTION
**Pull request by daryakoval**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/45613)

## Description
wrapper script for panorama-security-policy-match command

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [x] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/PAN-OS/Scripts/PanoramaSecurityPolicyMatchWrapper/test_data/basescript-dummy.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (dantavori) [inline Packs/PAN-OS/Scripts/PanoramaSecurityPolicyMatchWrapper/test_data/basescript-dummy.json:1]: remove if unused

File: Packs/PAN-OS/Scripts/PanoramaSecurityPolicyMatchWrapper/PanoramaSecurityPolicyMatchWrapper.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+- contextPath: Panorama.SecurityPolicyMatch.Rules.Source
+  description: The matching rule source.
+  type: String
+- contextPath: Panorama.SecurityPolicyMatch.Rules.To
+  description: The matching rule to zone.
+  type: String
+script: '-'
+system: false
+tags:
+- basescript
```
Comment (dantavori) [inline Packs/PAN-OS/Scripts/PanoramaSecurityPolicyMatchWrapper/PanoramaSecurityPolicyMatchWrapper.yml:68]: did we add this by design?

File: Packs/PAN-OS/Scripts/PanoramaSecurityPolicyMatchWrapper/PanoramaSecurityPolicyMatchWrapper.yml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+timeout: '0'
```
Comment (dantavori) [inline Packs/PAN-OS/Scripts/PanoramaSecurityPolicyMatchWrapper/PanoramaSecurityPolicyMatchWrapper.yml:69]: why 0?

**Review by dantavori — CHANGES_REQUESTED** [review]

**Review by dantavori — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@dantavori Doc review completed.

**Pull request merged**